Support language for VGH-833 (#13567)
hashicorp/vagrant: website/content/intro/support.mdx

---
layout: intro
page_title: Vagrant Support
description: "Find Vagrant & HCP Vagrant Support"
---
# Contacting Support

~> **Note**: You will need a Hashicorp Zendesk account to submit a support ticket. You may create an account [here](https://hashicorp.zendesk.com/auth/v2/login/registration)
or when prompted by selecting the `New to Hashicorp? Sign up` link at the bottom of the sign in widget.

To submit a support ticket:

> 1. Go to our [Support Page](https://support.hashicorp.com/hc/en-us). You may find the answers you need by searching our support knowledge base, but if not select `Open a new ticket`.

Reviewer: ```suggestion
1. Navigate to the HashiCorp [Support Page](https://support.hashicorp.com/hc/en-us). You may find the answers you need by searching our support knowledge base, but if not select `Open a new ticket`.
```

2. To submit a new support request select `Other / No Contract` and after reading the SLA disclaimer click `I still want to submit a ticket`.

> 3. You will then be prompted to log in to a Hashicorp specific zendesk. If you do not have an account you can create one by clicking the `Sign up` link at the bottom of the sign in widget.

Reviewer: ```suggestion
3. You will then be prompted to log-in to a HashiCorp specific Zendesk. If you do not have an account you can create one by clicking the `Sign up` link below the sign-in.
```
